subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/SentryConsumer.kt

```diff
@@ -1,97 +0,0 @@
-@file:Suppress("RemoveCurlyBracesFromTemplate")
-
-package com.avito.android.plugin.build_metrics
-
-import com.avito.android.gradle.metric.AbstractMetricsConsumer
-import com.avito.android.gradle.profile.BuildProfile
-import com.avito.android.sentry.sentry
-import com.avito.kotlin.dsl.getOptionalStringProperty
-import com.avito.logger.Logger
-import com.avito.logger.LoggerFactory
-import com.avito.logger.create
-import io.sentry.connection.EventSendCallback
-import io.sentry.event.Event
-import org.gradle.BuildResult
-import org.gradle.api.Project
-
-// todo вынести в фабрику знания про project
-internal class SentryConsumer(
```

Comment:
Not used for a long time. 
Maybe we will add a replacement later. I'll decide while troubleshooting network errors.